src/ORM/EagerLoader.php

```diff
@@ -27,6 +27,8 @@
  * for a table once a query is provided and delegates the job of creating the
  * required joins and decorating the results so that those associations can be
  * part of the result set.
+ *
+ * @internal
```

Comment:
If we can't mark this internal, the function I'm looking to modify is `setMatching()` and related code. I don't know that `contain()` needs to change.